assets/components/errorMessage/errorMessage.jsx

```diff
@@ -10,16 +10,24 @@ import Svg from 'components/svg/svg';
 
 type PropTypes = {
   message: string,
+  id?: string,
 };
 
 
 // ----- Component ----- //
 
-
 export default function ErrorMessage(props: PropTypes) {
   return (
-    <div className="component-error-message">
+    <div id={props.id || 'validation-error'} className="component-error-message">
```

Comment:
If `props.id` is null, the id attribute won't be rendered, which is I think what we want in this situation, so we probably don't need the 'validation-error' fallback id.